window_manager/include/core/RWindow.hpp

```diff
@@ -55,26 +65,28 @@ class XWWindow {
 
     [[nodiscard]] const std::shared_ptr<rendering::RenderingContext> &getRenderingContext() const;
 
+#ifdef _WIN32
+    [[nodiscard]] HWND getWindowHandle() const;
+    [[nodiscard]] HDC getGraphicContext() const;
```

Comment:
Refactor with X11 version, you know how.

Reply:
Done